src/polygon_mesh/mesh_factory.cpp

```
@@ -130,12 +128,12 @@ namespace plateau::polygonMesh {
         void findAllPolygonsInGeometry(
             const Geometry& geom, std::list<const citygml::Polygon*>& polygons,
             const unsigned lod, long long& out_vertices_count,
-            const std::vector<plateau::geometry::Extent> extents) {
+            const std::vector<plateau::geometry::Extent> extents, const MeshExtractOptions& options) {
```

Cppcheck (2.10-2)

[performance] Function parameter 'extents' should be passed by const reference.

(passedByValue)